ansible/configs/ansible-multitier-infra/files/hosts_template.j2

```diff
@@ -1,13 +1,30 @@
-{% for group in inventory_groups %}
+[frontends]
+{% for host in groups['frontends'] %}
+{% if cloud_provider == 'ec2' %}
+{{ host }}
+{% elif cloud_provider == 'osp' %}
+{{ host }}.example.com 
+{% endif %}
+{% endfor %}
+
+[apps]
+{% for host in groups['apps']  %}
+
```

Comment:
it looks like these two lines are unwanted .

Reply:
fixed